### DISCUSSION TITLE
Mosyle AV flagging the package as malware

### DISCUSSION THREAD

**Discussion by jiggie914** [ConsenSys-archive/truffle#5642] [Security]
We are using mosyle and its flagging your tools as malware
<img width="689" alt="screenshot_2022-10-24_at_11 20 42" src="https://user-images.githubusercontent.com/10282178/197827006-13f6be17-5e4e-4597-b734-35ea0e8afb65.png">

**Comment by kevinbluer** [comment]
Hey @jiggie914, thanks so much for heads up! We're looking into and will circle back shortly 👍
